server/repository/src/db_diesel/purchase_order_line.rs

```diff
@@ -11,20 +11,20 @@ use crate::{
 };
 
 use diesel::{
-    dsl::{InnerJoin, IntoBoxed},
+    dsl::{InnerJoin, IntoBoxed, LeftJoin},
     prelude::*,
 };
 
 type PurchaseOrderLineJoin = (
     PurchaseOrderLineRow,
-    (ItemLinkRow, ItemRow),
+    Option<(ItemLinkRow, ItemRow)>,
```

Comment:
Realised here that while the item_link_id was optional, we couldn't query results unless it was present.

I wonder if this is the right pattern to use?





Alternatively we could make it so that item_link_id is a not nullable field, and just use innerJoin